newIDE/electron-app/app/main.js

```diff
@@ -46,17 +49,18 @@ const args = parseArgs(process.argv.slice(isDev ? 2 : 1), {
 // See registerGdideProtocol (used for HTML modules support)
 protocol.registerSchemesAsPrivileged([{ scheme: 'gdide' }]);
 
+// TODO: Linked issue is resolved in v10.1.4+. Set to true (or remove) and check if stuff works
 // Should be set to true, which will be the default value in future Electron
 // versions, but then causes an issue on Windows where the `fs` module stops
 // working in the renderer process.
 // See https://github.com/electron/electron/issues/22119
 // For now, disable this as we rely heavily on `fs` in the renderer process.
-app.allowRendererProcessReuse = false;
+app.allowRendererProcessReuse = true;
```

Comment:
I think I committed this by mistake - this flag will be removed in v14 so I should've handled it later. Also, I haven't checked if it works on Windows on v10.4.7 yet (TODO)

Reply:
Feel free to go ahead with the upgrade and consider this is not needed anymore, as we want to land on a version that is at least Electron 14 I think :) 

So you can test on Windows once you have upgraded to a "sufficiently recent" version.
In other words, I don't want you to lose time trying to understand a bug that would have been fixed by a newer Electron version ;)

Reply:
Checked on Electron v11 on Windows, `fs` module is working (can open local project). Resolving this comment now.

Reply:
I believe it was working but failing after you reload one or twice the development version by pressing Ctrl+R. But I think they resolved this at some point according to the issue on GitHub :)